src/jobs/test.yml

```diff
@@ -35,6 +35,10 @@ parameters:
       Optionally supply a custom package installation command, with any additional flags needed.
     type: string
     default: ""
+  run-step-name:
+    description: Title of the run step to be shown in the CircleCI UI
+    type: string
+    default: Run <<parameters.pkg-manager>> tests
```

Comment:
Unfortunately, this doesn't work, as shown here on line 221: https://app.circleci.com/pipelines/github/CircleCI-Public/node-orb/622/config

Reply:
not sure you sent the right link, that one doesn't even have a line 221

Reply:
You'll have to click over to the "Compiled" tab in the top right :)

Reply:
interestingly it didn't just spit the string, it actually evaluated the variable to ... nothing :P